core/sql/cli/Cli.cpp

```diff
@@ -8325,19 +8325,15 @@ Lng32 SQLCLI_RegisterQuery(CliGlobals *cliGlobals,
   StatsGlobals *statsGlobals = cliGlobals->getStatsGlobals();
   if (statsGlobals == NULL)
     return retcode;
-  short error;
-  short savedPriority, savedStopMode;
+  int error;
   error = statsGlobals->getStatsSemaphore(cliGlobals->getSemId(),
-                 cliGlobals->myPin(), 
-                savedPriority, savedStopMode, FALSE /*shouldTimeout*/);
-  ex_assert(error == 0, "getStatsSemaphore() returned an error");
-
+                 cliGlobals->myPin());
```

Comment:
Why do we ignore the error?

Reply:
Any error would have asserted in StatsGlobals::getStatsSemaphore() method itself